Makefile

```
@@ -43,6 +53,25 @@ clean-db:  ## remove LMS databases
 	$(COMPOSE) rm edx_mongodb edx_mysql edx_redis 
 .PHONY: clean-db
 
+install-tests: ## install tests dependencies
```

As front-end dependencies are the only requirements, we can make this rule name shorter?
```suggestion
install: ## install tests dependencies
```